metashare/api/views.py

```diff
@@ -97,6 +98,21 @@ class TaskViewSet(viewsets.ModelViewSet):
     filter_backends = (DjangoFilterBackend,)
     filterset_class = TaskFilter
 
+    @action(detail=True, methods=["POST"])
```

Comment:
I don't really care practically, but in some ways it really feels like this belongs on the `org`, not the `task`. 🤷‍♂